release/ray_release/test_automation/ci_state_machine.py

```diff
@@ -46,24 +63,39 @@ def _create_github_issue(self) -> None:
             "triage",
             self.test.get_oncall(),
         ]
+        recent_failures = [
+            result for result in self.test_results if result.is_failing()
+        ][:3]
```

Comment:
what does this `[:3]` do? why it is 3?

Reply:
let me put in a constant, the goal is to provide a few instances so test owner can debug them